slsim/Sources/Supernovae/supernovae_pop.py

```diff
@@ -11,7 +11,7 @@ def calculate_star_formation_rate(z):
     """Calculates the cosmic star formation rate. (Eq 13 - Oguri and Marshall 2010)
 
     :param z: redshift (z>=0)
-    :return: cosmic star formation rate in [(h)yr^(-1)Mpc^(-3)]
+    :return: cosmic star formation rate in [(M_sol)yr^(-1)Mpc^(-3)]
```

Comment:
are you sure there is no 'little h' in there?

Reply:
You're right I shouldn't have deleted it, just added the solar mass.